pkgs/by-name/rs/rsop/package.nix

```
@@ -0,0 +1,43 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitea,
+  pkg-config,
+  pcsclite,
+  nix-update-script,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "rsop";
+  version = "0.3.9";
+
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "heiko";
+    repo = pname;
+    rev = "rsop/v${version}";
+    hash = "sha256-K69vyZFaVvZj4yLaV/zQYoItvcTDuFR4mdmMcfl1UDA=";
+  };
+
+  cargoHash = "sha256-DJVgnfPpXf8hGX6Dv6W8GzqspMEFZHc2/Fkn1MZRXd0=";
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [ pcsclite ];
+
+  passthru = {
```

A basic test might be nice
```suggestion
  passthru = {
    tests.version = testers.testVersion {
      command = "rsop version";
      package = rsop;
    };
```